### PR TITLE
document deployment

### PR DESCRIPTION
**Pull request by mccalluc**
I'm not confident that this even belongs here, but wanted to get the steps down while they were fresh in my mind. It might be better to find a place for this in on-boarding... and maybe there is, and I just missed it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,10 +51,25 @@ To stop database services:
...
+
+Copy the ID that looks like `U0123456789`, and make a PR to update the [list of allowed users](https://github.com/pulibrary/pulbot/blob/main/scripts/listener_middleware.coffee).
+A peer who already has privs should merge your PR, redeply pulbot itself with `pulbot deploy pulbot`.
+When the pulbot redeploy is complete you'll be able to complete your deploy to staging.
+
+PDC applications are only available on campus, or through the VPN.
 
 To create a tagged release use the [steps in the RDSS handbook](https://github.com/pulibrary/rdss-handbook/blob/main/release_process.md)
 
+You can specify the tagged version in deployments to production: `pulbot deploy pdc_describe/v1.2.3 to production`
```
Comment (mccalluc) [inline README.md:71]:
```suggestion
You should specify the tagged version in deployments to production: `pulbot deploy pdc_describe/v1.2.3 to production`
```

**Review by hectorcorrea — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -51,10 +51,25 @@ To stop database services:
...
+- Some PRs will be deployed to `staging` before merge to confirm they work in that environment.
+- Most PRs (beyond documentation changes) should be deployed to `production` after merge to get features out to users.
+- Many redeployments will also be new versioned releases, but a release is not strictly necessary for every redeployment.
+
+Deployments are done with `pulbot`: You'll need to be added to the `#robots` channel in PULibrary slack.
+Once you're added, try `pulbot deploy pdc_describe to staging`: You'll get an error message like:
+> I'm sorry, @UserName (U0123456789), but you don't have access to do that.
+
+Copy the ID that looks like `U0123456789`, and make a PR to update the [list of allowed users](https://github.com/pulibrary/pulbot/blob/main/scripts/listener_middleware.coffee).
+A peer who already has privs should merge your PR, redeply pulbot itself with `pulbot deploy pulbot`.
```
Comment (mccalluc) [inline README.md:64]:
```suggestion
A peer who already has privs should merge your PR, and redeploy pulbot itself with `pulbot deploy pulbot`.
```

**Comment by bess** [comment]
@mccalluc This is great documentation, but I think it belongs more in the rdss handbook. Maybe here? https://github.com/pulibrary/rdss-handbook/blob/main/release_process.md
We could change that page to be more inclusive, maybe "Deploying and Release Process"?

**Review by hectorcorrea — DISMISSED** [review]

**Comment by mccalluc** [comment]
Thanks: I've moved most of these changes to https://github.com/pulibrary/rdss-handbook/pull/88 -- I'll make a link to those docs here.

**Review by hectorcorrea — APPROVED** [review]

**Pull request merged**